build.gradle.kts

```diff
@@ -159,6 +162,12 @@ subprojects {
                     }
                 }
             }
+            repositories {
+                maven {
+                    name = "sonatype-nexus-staging"
```

Comment:
Are we keeping staging after all?

Reply:
Yes because we need to say where to deploy maven central artifacts to (See http://www.albertgao.xyz/2018/01/18/how-to-publish-artifact-to-maven-central-via-gradle/ )

Reply:
:+1